packages/api/src/api/user.ts

```
@@ -0,0 +1,183 @@
+   * @request POST:/user/createWithList
+   */
+
+  async createUsersWithListInput(payload: User[]) {
+    try {
+      const method = "post";
+      const endpoint = "/user/createWithList";
+      const options = payload;
+
+      return await apiInstance[method](endpoint, options);
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)